content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,203 @@
 
-  <!-- Load our React component. -->
+  <!-- A React komponensünk betöltése. -->
   <script src="like_button.js"></script>
 
 </body>
 ```
 
-The first two tags load React. The third one will load your component code.
+A két első címke a Reactet töltik be. A harmadik a komponensed kódját fogja betölteni.
 
-### Step 3: Create a React Component {#step-3-create-a-react-component}
+### Harmadik lépés: Készíts egy React komponenst {#step-3-create-a-react-component}
 
-Create a file called `like_button.js` next to your HTML page.
+Készíts egy fájlt a HTML oldalad mellett és nevezd el `like_button.js`-nek.
 
-Open **[this starter code](https://gist.github.com/gaearon/0b180827c190fe4fd98b4c7f570ea4a8/raw/b9157ce933c79a4559d2aa9ff3372668cce48de7/LikeButton.js)** and paste it into the file you created.
+Nyisd meg **[ezt a kezdő kódot](https://gist.github.com/gaearon/0b180827c190fe4fd98b4c7f570ea4a8/raw/b9157ce933c79a4559d2aa9ff3372668cce48de7/LikeButton.js)** és másold be a tartalmát a fájlba amit készítettél.
 
->Tip
+>Tipp
 >
->This code defines a React component called `LikeButton`. Don't worry if you don't understand it yet -- we'll cover the building blocks of React later in our [hands-on tutorial](/tutorial/tutorial.html) and [main concepts guide](/docs/hello-world.html). For now, let's just get it showing on the screen!
+> Ez a kód egy `LikeButton` nevű React komponenst definiál. Ne aggódj ha még nem érted -- később át fogjuk venni a React építőelemeit a [gyakorlati tutoriálunkban](/tutorial/tutorial.html) és a [főbb koncepciók útmutatónkban](/docs/hello-world.html). Egyenlőre elég ha meggyőződünk róla, hogy megjelenik a képernyőn!
 
-After **[the starter code](https://gist.github.com/gaearon/0b180827c190fe4fd98b4c7f570ea4a8/raw/b9157ce933c79a4559d2aa9ff3372668cce48de7/LikeButton.js)**, add two lines to the bottom of `like_button.js`:
+
+Adj hozzá két sort a `like_button.js` alján **[a kezdő kód](https://gist.github.com/gaearon/0b180827c190fe4fd98b4c7f570ea4a8/raw/b9157ce933c79a4559d2aa9ff3372668cce48de7/LikeButton.js)** után:
 
 ```js{3,4}
-// ... the starter code you pasted ...
+// ... a kezdő kód amit bemásoltál ...
 
 const domContainer = document.querySelector('#like_button_container');
 ReactDOM.render(e(LikeButton), domContainer);
 ```
 
-These two lines of code find the `<div>` we added to our HTML in the first step, and then display our "Like" button React component inside of it. 
+Ez a két sor kód megtalálja a `<div>`-et amit a HTML-hez adtunk az első lépésben, és megjeleníti benne a "Tetszik" React komponens gombunkat.
 
-### That's It! {#thats-it}
+### És ennyi! {#thats-it}
 
-There is no step four. **You have just added the first React component to your website.**
+Nincs negyedik lépés. **Ezzel hozzáadtad az első React komponenst a weblapodhoz.**
 
-Check out the next sections for more tips on integrating React.
+Még több React integrálási tippért nézd meg a következő fejezeteket.
```

Comment:
I think this refers to the next sections on this same page and not other pages? Not sure what the correct term for that is